controllers/status/status.go

```diff
@@ -190,6 +197,11 @@ func SetComponentCondition(conditions *[]conditionsv1.Condition, component strin
 
 // RemoveComponentCondition remove Condition of giving component.
 func RemoveComponentCondition(conditions *[]conditionsv1.Condition, component string) {
-	condType := component + ReadySuffix
+	var condType string
+	if component == trustyai.ComponentName {
+		condType = component + "Deprecated"
+	} else {
```

Comment:
think we should not set this, 
eventually this status will be removed so not leave a TrustyAIDeprecated in the condition.

Reply:
Okay I will update this